frontend/appflowy_flutter/lib/plugins/base/drag_handler.dart

```diff
@@ -10,7 +10,7 @@ class DragHandler extends StatelessWidget {
     return Container(
       height: 4,
       width: 40,
-      margin: const EdgeInsets.symmetric(vertical: 8),
+      margin: const EdgeInsets.only(top: 4),
```

Comment:
Changing this also requires corresponding changes in many grid mobile sheets.

Reply:
Yes. I reverted it.